public/pages/Monitors/containers/Monitors/utils/helpers.js

```diff
@@ -24,6 +25,7 @@ export function getURLQueryParams(location) {
     sortField,
     sortDirection,
     state,
+    dataSourceId: dataSourceId === undefined ? undefined : dataSourceId,
```

Comment:
Nitpick:
Do we need this check? In both cases, `dataSourceId` keeps its value. I.e., if it's undefined, it stays undefined; and if it's defined, it keeps its value.

Reply:
addressed